DataminrPulse/DataminrPulse.manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+    "author": "Dataminr",
+    "categories": [
+        "Compliance and Security",
+        "IT Infrastructure",
+        "Sumo Logic Certified"
+    ],
+    "communityURL": "https://support.sumologic.com/hc/en-us/community/topics/200263058-Applications-and-Integrations",
+    "description": "Dataminr Pulse\u00e2\u20ac\u2122s AI-powered real-time intelligence brings faster detection and response into Sumo Logic Cloud Security Analytics. Learn about threats as they unfold to prioritize and manage risk, and protect your physical and digital assets in real time. Dataminr Pulse for Cyber Risk easily fits into your Sumo Logic Cloud Security Analytics workflows, enabling rapid identification and mitigation of emerging threats so you can deliver faster time to detection and response. The global leader in AI for risk detection since 2009, Dataminr Pulse is relied on by two-thirds of Fortune 100 companies to inform their physical, cyber, and converged security operations.",
+    "helpURL": "https://help.sumologic.com/?cid=TODO",
```

Comment:
We can also point the helpURL to the github README

Reply:
Updated the `helpURL` with GitHub README.